model/plugin_constants.go

```diff
@@ -10,4 +10,6 @@ const (
 	PluginIdCalls         = "com.mattermost.calls"
 	PluginIdNPS           = "com.mattermost.nps"
 	PluginIdChannelExport = "com.mattermost.plugin-channel-export"
+	PluginIdJenkins       = "jenkins"
+	PluginIdGithub        = "github"
```

Comment:
It seems this list of plugin ids are related to the ones that we have prepackaged. If the plugin ids for Jenkis and Github are only for testing purposes, I would move this directly to the test files and avoid confusion on why only these two other specific plugins are defined here.